scripts/optimize-assets.js

```diff
@@ -59,23 +59,84 @@ function get_relative_path(file) {
 }
 
 async function main() {
+    console.log(
+        'This script runs for ~5 mins. It runs silently if all files are already optimized.'
+    );
+
     for (const file of walk_directory(join(__dirname, '../static'))) {
         const relative_path = get_relative_path(file);
         if (!is_image(file)) continue;
         if (exceptions.some((exception) => relative_path.startsWith(exception))) continue;
 
-        console.log(relative_path);
         const image = sharp(file);
-        const size_before = (await image.toBuffer()).length;
-        const meta = await image.metadata();
+
+        let meta;
+        try {
+            meta = await image.metadata();
```

Comment:
If this throws, sharp doesnt like file. Good enough reason to consider file bad.